internal/services/formatters/csharp/dotnet_cli/formatter.go

```diff
@@ -156,7 +166,21 @@ func (f *Formatter) newVulnerability(dependency *dotnetDependency, projectSubPat
 		Line:         line,
 		Severity:     dependency.getSeverity(),
 	}
-	return f.SetCommitAuthor(vulnHash.Bind(vuln))
+	return f.SetCommitAuthor(vulnHash.Bind(vuln)), nil
+}
+
+func (f *Formatter) getCodeFilePathAndLine(dependency *dotnetDependency,
+	projectSubPath string) (string, string, string, error,
+) {
+	code, filePath, line, err := file.GetDependencyCodeFilepathAndLine(
+		f.GetConfigProjectPath(), projectSubPath, dependency.Name, CsProjExt,
+	)
+	if err != nil {
+		f.SetAnalysisError(err, tools.DotnetCli, err.Error(), "")
+		logger.LogErrorWithLevel(messages.MsgErrorGetDependencyCodeFilepathAndLine, err)
```

Comment:
I think that these lines should be moved to `addVulnIntoAnalysis` error check.

I think that would be better because the addVulnIntoAnalysis function is the function that call the functions that return an error, so the error check would make more sense there.